lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -22,6 +22,12 @@ function dosomething_organ_donation_menu() {
     'type' => MENU_CALLBACK,
   ];
 
+  $items['organ-donation/post-registration'] = [
+    'page callback' => 'dosomething_organ_donation_post_registration',
+    'access callback' => 'user_is_logged_in',
```

Comment:
I missed this on the last PR, the permissions should be only enabled for admin users.